### PR TITLE
fix: add retryTask for all exceptions

### PR DESCRIPTION
**Pull request by tikhanovichA**
For some cases related to remote publications, webhook can be failed because delivery is not ready yet. 
We can manage all those retries using `retryMax` option in the webhookRegistry.conf.php if it will be needed.

We need also backport it to v51.0.3

### CODE REVIEW

**Comment by andreluizmachado** [comment]
Hey, can we also fix PHP CS mentioned on [CIs](https://github.com/oat-sa/tao-core/actions/runs/4304532634/jobs/7505695016)?

File: models/classes/webhooks/WebhookTaskServiceInterface.php [hunk 1]
```diff
@@ -39,6 +39,7 @@ public function linkTaskToQueue($queueName);
     /**
```
Comment (pnal) [inline models/classes/webhooks/WebhookTaskServiceInterface.php:39]: `public const SERVICE_ID = 'tao/webhookTaskService';`
on line 30

**Review by pnal — CHANGES_REQUESTED** [review]
Please fix a CI CS warning

**Review by wazelin — CHANGES_REQUESTED** [review]
This looks alright-ish, but potentially dangerous.

Can a more specific exception be added instead to cover the case you're trying to here? If not, consider this approved once the CI is back to green again.

**Pull request closed without merging**